.vscode/settings.json

```diff
@@ -1,5 +1,3 @@
 {
-	"cSpell.words": [
-		"reqs"
-	]
+  "cSpell.words": ["reqs"]
```

Comment:
Not sure why this ended up in here, but I'll remove it on the next push after feedback.